content/en-us/reference/engine/classes/TextChannel.yaml

```diff
@@ -183,24 +183,23 @@ callbacks:
     thread_safety: Unsafe
   - name: TextChannel.ShouldDeliverCallback
     summary: |
-      Called when `Class.TextChannel` is receiving an incoming message to
-      determine whether or not it should be delivered to other clients.
+      Called for each client when `Class.TextChannel` is receiving an incoming message to determine whether or not it should be delivered to that client.
     description: |
-      Called when `Class.TextChannel` is receiving an incoming message to
-      determine whether or not to deliver the message to other clients. Can only
-      be defined on the server.
+      Called for each client when `Class.TextChannel` is receiving an incoming message to determine whether or not it should be delivered to that client. Can only be defined on the server.
 
-      Once defined, this callback needs to return a truthy value (true, 1,
-      "hello") to deliver the message to other clients. If the callback returns
-      anything else (including nil), the message can't be delivered. The sender
-      will still see the message even if the message can't be delivered.
+      Once defined, this callback needs to return a truthy value such as `true`, `1`,
+      or `"hello"` to deliver the message to said client. If the callback returns
+      anything else (including `nil`), the message won't be delivered to that client, although the
+      sender will see the message regardless.
+      
+      The sender can be referenced by 'Class.TextChatMessage.TextSource'.
     code_samples: []
     parameters:
       - name: message
         type: TextChatMessage
         default:
         summary: ''
-      - name: textSource
+      - name: targetTextSource
```

Comment:
```suggestion
      - name: textSource
```

Reply:
Just FYI, only summaries, descriptions, and deprecation messages can be changed for classes and enums. Other changes will be automatically overwritten.